website/client/components/groups/group.vue

```diff
@@ -418,6 +432,9 @@ export default {
       if (!this.group.challenges) return false;
       return this.group.challenges.length === 0;
     },
+    showNoNotificationsMessage () {
+      return this.group.memberCount > 5000;
```

Comment:
we should use the constant defined in the common code instead of the number directly, $store.state.constants.LARGE_GROUP_COUNT_MESSAGE_CUTOFF